common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -1111,6 +1111,12 @@
     ],
     "sqlState" : "42K03"
   },
+  "DATA_SOURCE_UNSUPPORTED_DML_OPERATION" : {
```

Comment:
if we removed DML from message, let's remove it from the condition name.

Reply:
I want to have it just exception dedicated to DML operations, if it is needed I can revert DML in message? Serge suggested me to do that

Reply:
The message should be in sync with the name. So yes, I implied to remove it from both.